src/components/ProtestForm/ProtestForm.js

```diff
@@ -84,6 +86,9 @@ function ProtestForm({
     nearbyProtests();
   }, [coordinatesUpdater]);
 
+  const onCancel = () => {
+    history.goBack();
```

Comment:
Pardon, I take back my words about the confirmation message.

Add a `confirm` dialog that says "המידע שהזנת לא יישמר. לחזור אחורה?"

Reply:
I was prepared for that :smiley: